AStarAI/src/main/java/com/github/sef24sp4/astarai/caching/IPathCaching.java

```diff
@@ -0,0 +1,11 @@
+package com.github.sef24sp4.astarai.caching;
+
+import com.github.sef24sp4.common.entities.ICollidableEntity;
+import com.github.sef24sp4.common.vector.IVector;
+
+public interface IPathCaching {
+
+	public boolean flush();
+
+	public IVector getNextCoordinates(ICollidableEntity entity, final IVector targetCoordinates);
```

Comment:
Change method spec so it only takes `IVector`. 
Parameter `entity` is redundant, as caches are constructed with an entity.